### PR TITLE
app: server uncompressed WASM to webkit browsers

### LINKED ISSUE #505: 0.4.2 won't start in Safari (docker image)
I upgraded from 0.4.1 to 0.4.2, after this LDAP and Web Interface won't start for me. Downgrading to 0.4.1 and everything is working again.
Running this using docker compose, right now using the "image: nitnelave/lldap:v0.4.1".

**Starting 0.4.1 is showing this logging:**
Setup permissions..
Starting lldap..

Loading configuration from /data/lldap_config.toml
2023-03-27T17:42:15.019508521+00:00 INFO     set_up_server [ 2.12ms | 100.00% ]
2023-03-27T17:42:15.019538644+00:00 INFO     ┝━ ｉ [info]: Starting LLDAP version 0.4.1
2023-03-27T17:42:15.030860201+00:00 INFO     ┝━ ｉ [info]: Starting the LDAP server on port 3890
2023-03-27T17:42:15.034345442+00:00 INFO     ┕━ ｉ [info]: Starting the API/web server on port 17170
2023-03-27T17:42:15.034719186+00:00 INFO     ｉ [info]: Starting 1 workers | log.target: "actix_server::builder" | log.module_path: "actix_server::builder" | log.file: "/__w/lldap/lldap/${GITHUB_WORKSPACE}/.cargo/registry/src/github.com-1ecc6299db9ec823/actix-server-2.0.0-beta.5/src/builder.rs" | log.line: 281
2023-03-27T17:42:15.035099867+00:00 INFO     ｉ [info]: Starting "ldap" service on 0.0.0.0:3890 | log.target: "actix_server::builder" | log.module_path: "actix_server::builder" | log.file: "/__w/lldap/lldap/${GITHUB_WORKSPACE}/.cargo/registry/src/github.com-1ecc6299db9ec823/actix-server-2.0.0-beta.5/src/builder.rs" | log.line: 296
2023-03-27T17:42:15.035268249+00:00 INFO     ｉ [info]: Starting "http" service on 0.0.0.0:17170 | log.target: "actix_server::builder" | log.module_path: "actix_server::builder" | log.file: "/__w/lldap/lldap/${GITHUB_WORKSPACE}/.cargo/registry/src/github.com-1ecc6299db9ec823/actix-server-2.0.0-beta.5/src/builder.rs" | log.line: 296
2023-03-27T17:42:15.035494339+00:00 INFO     ｉ [info]: DB Cleanup Cron started
2023-03-27T17:43:00.525304561+00:00 INFO     HTTP request [ 16.2µs | 100.00% ]
2023-03-27T17:43:00.525311012+00:00 INFO     ┝━ ｉ [info]:  | uri: /
2023-03-27T17:43:00.525332563+00:00 INFO     ┕━ ｉ [info]:  | status_code: 404
2023-03-27T17:43:00.704929560+00:00 INFO     HTTP request [ 155µs | 100.00% ]
2023-03-27T17:43:00.704935571+00:00 INFO     ┝━ ｉ [info]:  | uri: /
2023-03-27T17:43:00.705091946+00:00 INFO     ┕━ ｉ [info]:  | status_code: 200

**Starting 0.4.2 is only showing this logging:**
ldap-docker-lldap-1  | > Setup permissions..
ldap-docker-lldap-1  | > Starting lldap..
ldap-docker-lldap-1  | 
ldap-docker-lldap-1  | Loading configuration from /data/lldap_config.toml
ldap-docker-lldap-1  | 2023-03-27T17:38:42.800325707+00:00  INFO     set_up_server [ 1.62ms | 100.00% ]
ldap-docker-lldap-1  | 2023-03-27T17:38:42.800334827+00:00  INFO     ┝━ ｉ [info]: Starting LLDAP version 0.4.2
ldap-docker-lldap-1  | 2023-03-27T17:38:42.803710790+00:00  INFO     ┝━ ｉ [info]: Starting the LDAP server on port 3890
ldap-docker-lldap-1  | 2023-03-27T17:38:42.804550144+00:00  INFO     ┕━ ｉ [info]: Starting the API/web server on port 17170
ldap-docker-lldap-1  | 2023-03-27T17:38:42.805081434+00:00  INFO     ｉ [info]: starting 1 workers
ldap-docker-lldap-1  | 2023-03-27T17:38:42.805423622+00:00  INFO     ｉ [info]: Actix runtime found; starting in Actix runtime
ldap-docker-lldap-1  | 2023-03-27T17:38:42.806473704+00:00  INFO     ｉ [info]: DB Cleanup Cron started
ldap-docker-lldap-1  | 2023-03-27T17:39:12.852146791+00:00  INFO     LDAP session [ 134µs | 58.99% / 100.00% ]
ldap-docker-lldap-1  | 2023-03-27T17:39:12.852210127+00:00  INFO     ┕━ LDAP request [ 54.9µs | 41.01% ]

Unfortunately I don't have any more logging. Checked the latest changelog but didn't see any major changes which would need a change in config.

### PR DESCRIPTION
**Pull request by nitnelave**
Fixes #505

### CODE REVIEW

**Comment by nitnelave** [comment]
@pixelrazor could you have a look at this PR? Does it look okay?

File: server/src/infra/tcp_server.rs [hunk 1]
```diff
@@ -110,12 +114,9 @@ fn http_config<Backend>(
             .configure(super::graphql::api::configure_endpoint::<Backend>),
     )
     .service(
-        web::resource("/pkg/lldap_app_bg.wasm").route(
-            web::route()
-                .wrap(middleware::Compress::default())
```
Comment (nitnelave) [inline server/src/infra/tcp_server.rs:115]: I think the compress middleware was not doing anything since I was serving an already-compressed file.

**Comment by nitnelave** [comment]
I tried it on windows (firefox), android (firefox, chrome) and iOS (safari, firefox, chrome), but not MacOS.

**Review by pixelrazor — APPROVED** [review]
LGTM, I agree that we can remove that middleware (already compressed, and then not compressing the uncompressed to ensure WebKit likes it)

**Pull request merged**